mne/time_frequency/tfr.py

```diff
@@ -649,18 +649,27 @@ def plot(self, picks=None, baseline=None, mode='mean', tmin=None,
         dB : bool
             If True, 20*log10 is applied to the data to get dB.
         colorbar : bool
-            If true, colorbar will be added to the plot
+            If true, colorbar will be added to the plot. For user defined axes,
+            the colorbar cannot be drawn. Defaults to True.
         show : bool
             Call pyplot.show() at the end.
         title : str | None
             String for title. Defaults to None (blank/no title).
         verbose : bool, str, int, or None
             If not None, override default verbose level (see mne.verbose).
+        axes : instance of Axes | list | None
+            The axes to plot to. If list, the list must be a list of Axes of
+            the same length as the number of channels. If instance of Axes,
+            there must be only one channel plotted.
 
         Returns
         -------
         fig : matplotlib.figure.Figure
             The figure containing the topography.
+
+        Notes
+        -----
+        .. versionadded:: 0.10.0
```

Comment:
I think so, in which case this directive should go in the docstring for the `axes` parameter (and check that docs look okay)